spec/factories/service.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :service, :class => "Ovirt::Service" do
+    initialize_with { new(:server => "", :username => "", :password => "") }
```

Comment:
FactoryGirl definitely works with non AR / hashes / ... e.g.: [stack overflow]

[stack overflow]: http://stackoverflow.com/questions/10032760/how-to-define-an-array-hash-in-factory-girl